tests/test_datasets.py

```diff
@@ -1,9 +1,15 @@
 import yaml
 import urllib.request
 import pytest
+from fake_useragent import UserAgent
 
 
-d = yaml.load(open("mir-datasets.yaml"))
+# get fake user agent
+ua = UserAgent()
+
+# open dataset file
+with open('mir-datasets.yaml', 'rb') as hdl:
```

Comment:
I would expect this to be `'r'`, rather than `'rb'`, but I'm guessing this works?